app/models.py

```diff
@@ -162,6 +163,12 @@ class AliasGeneratorEnum(EnumE):
     uuid = 2  # aliases are generated based on uuid
 
 
+class Hibp(db.Model, ModelMixin):
+    __tablename__ = "hibp"
+    id = db.Column(db.String(), primary_key=True)
```

Comment:
Ah this `id` field will conflict with the ModelMixin.id and different than the convention in the code to use integer for `id` column. I think we can use the `name` with a unique constraint here as in the previous commit. It's also easier to match with the response from HIBP.

Reply:
Well, it overrides the id from ModelMixin, I see no issues with the way I built it. The problem with making name a new constraint is that each breach has an unique name so it feels illogical for that to not be the primary key. Unless I'm missing something meaningful?

I can change it if you really want though, but I'm not truly convinced myself it would be cleaner.

Reply:
Hey a table can have a primary key and several unique constraints so database wise, it doesn't feel unnatural for me to have an `id` as primary key with a couple of unique constraints. I think the main benefit of having a default integer `id` in all tables is consistency as we know this column is going to be there in all tables :)

I don't like this field is called "name" though as it doesn't convey "uniqueness" but this is a HIBP choice ...

Reply:
Fixed in a08b0c05cc3d41c664dfab7aab10cb5addc948b7.